__docs__/wonder-blocks-search-field/search-field.stories.tsx

```diff
@@ -66,15 +70,26 @@ const Template = (args: any) => {
     };
 
     return (
-        <SearchField
-            {...args}
-            value={value}
-            onChange={handleChange}
-            onKeyDown={(e) => {
-                action("onKeyDown")(e);
-                handleKeyDown(e);
-            }}
-        />
+        <View>
+            <SearchField
+                {...args}
+                value={value}
+                onChange={handleChange}
+                onKeyDown={(e) => {
+                    action("onKeyDown")(e);
+                    handleKeyDown(e);
+                }}
+                onValidate={setErrorMessage}
+            />
+            {(errorMessage || args.error) && (
+                <>
+                    <Strut size={spacing.xSmall_8} />
```

Comment:
That makes sense! I wonder if it would be helpful to add a `gap` or `space` prop to `View`. What do you think?

Also, is it preferred we inline the aphrodite styles or create styles in a stylesheet for things like this? 
```
<View style={{ gap: spacing.xSmall_8 }} />
```
or
```
<View style={styles.container} />
...
const styles = StyleSheet.create({
  container: {
    gap: spacing.xSmall_8,
  }
});
```
Is one more performant? One nice thing about inlining the styles is that if the styling gets removed, it's removed! Unlike the other way, there's a chance we forget to remove the unused style from the stylesheet! This may also change depending on our css approach, curious if anyone has thoughts on this :)

Reply:
The inline approach sure reminds me of Tailwind! It puts the styles closer to their usage. One downside is the code is less portable, whereas classes can be centralized and reused.